lapce-proxy/src/plugin/mod.rs

```diff
@@ -805,6 +805,38 @@ impl PluginCatalogRpcHandler {
         );
     }
 
+    pub fn action_resolve(
+        &self,
+        item: CodeAction,
+        cb: impl FnOnce(Result<CodeAction, RpcError>) + Send + Clone + 'static,
+    ) {
+        let method = CodeActionResolveRequest::METHOD;
+        self.send_request_to_all_plugins(
```

Comment:
It would be better to include the plugin_id in the code action response, and then only send to the same plugin_id here.

Reply:
updated

Reply:
Excellent! Thanks.